core/pom.xml

```diff
@@ -119,6 +119,12 @@
             <artifactId>paimon-bundle</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.roaringbitmap</groupId>
+            <artifactId>RoaringBitmap</artifactId>
+            <version>0.9.25</version>
```

Comment:
I recommend managing the dependency in the `DependencyManagement` blocker in parent pom and store the version in properties.

Reply:
BTW, should we dependent on version 0.9.44 as Iceberg project do.